src/actions/go-to.ts

```diff
@@ -0,0 +1,22 @@
+import 'colors';
+import { Browser } from 'puppeteer';
+import Action from '../action.i';
+
+/**
+ * Opens a new window for a given browser and navigates to a specified url.
+ */
+
+export default class GoTo implements Action {
+  constructor() {}
+
+  // @ts-ignore
+  async execute(title: string, args: string[], outs: any[], context: any): Promise<any> {
+    var browser: Browser = context.browser;
```

Comment:
We can simplify the code if we store the `page` inside the context instead of the browser. We can design a way for parallel execution sometime later.

Reply:
Done